### PR TITLE
feat(settings): Hook up new settings screen, remove old one

### PR DESCRIPTION
**Pull request by jophish**
### Description

Completes [RET-1180](https://linear.app/valora/issue/RET-1180/%5Bsettings%5D-create-new-settings-screen). Hooks up the new settings screen to the rest of the app, and removes the old one from the codebase entirely.

### Test plan

Unit and manual tested. See video below:

https://github.com/user-attachments/assets/ec5e53ef-7d4a-42d0-9227-154f3432da8b

### Related issues

- Fixes [RET-1180](https://linear.app/valora/issue/RET-1180/%5Bsettings%5D-create-new-settings-screen).

### Backwards compatibility

Yes.

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/icons/GearIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Svg, { Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  size?: number
+  color?: string
+}
+
+const GearIcon = ({ size = 22, color = colors.black }: Props) => (
+  <Svg width={size} height={size} fill="none">
```
Comment (satish-ravi) [inline src/icons/GearIcon.tsx:11]: for size parameter to work, a viewBox also needs to be specified

File: src/navigator/Headers.tsx [hunk 2]
```diff
@@ -234,7 +234,7 @@ export const tabHeader: NativeStackNavigationOptions = {
   },
   headerLeft: () => (
     <View style={[styles.topElementsContainer, { marginLeft: Spacing.Tiny4 }]}>
-      <AccountCircleButton testID="WalletHome/AccountCircle" />
+      <SettingsGearButton testID="WalletHome/AccountCircle" />
```
Comment (kathaypacific) [inline src/navigator/Headers.tsx:237]:
```suggestion
      <SettingsGearButton testID="WalletHome/SettingsGearButton" />
```

File: src/icons/GearIcon.tsx (hunk 1 above)
Comment (kathaypacific) [reply src/icons/GearIcon.tsx:11]: +1

**Review by kathaypacific — APPROVED** [review]
LGTM, would be good to address the small comments before merging! 🚀

**Pull request merged**